canonical/identity-platform-admin-ui
# WD-15024 - feat: add user invite email template

## Changes

Added a template for the user verification email.
NOTE: the built template will be committed in an upcoming PR.

https://warthogs.atlassian.net/browse/WD-15024

## QA

- `cd ui/emails` and `make build`.
- Open `internal/mail/html/user-invite.html` in your browser and check that it looks like [the designs](https://www.figma.com/design/eFR98XGAliMnPUWeBtWzKI/Login-UI?node-id=1876-4176&node-type=frame&t=pfPtLZOs92A6W6EX-0) at desktop and mobile sizes.
